tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -126,15 +126,15 @@
     <comment>Dev drive size free</comment>
   </data>
   <data name="DevDriveInsightsCard.Description" xml:space="preserve">
-    <value>All things, Dev Drives, optimizations, etc.</value>
+    <value>All things Dev Drives, optimizations, etc.</value>
```

Comment:
"All things Dev Drives" seems awkward to me. Maybe "All things Dev Drive" ?

Reply:
@cinnamon-msft ?

Reply:
Yeah "All things Dev Drive" sounds good - I don't own this feature but I agree it sounds better. :)

Reply:
Updated.